tests/pipeline/snapshots/snap_test_example.py

```diff
@@ -25,8 +25,6 @@
                 'schedule': '0 12 * * *',
                 'streams': {
                     'brokers': 'http://k8kafka-cp-kafka-headless.kpops.svc.cluster.local:9092',
-                    'extraOutputTopics': {
```

Comment:
Is this the correct behavior? The `generate` output does not show the `extraOutputTopics`, `from`, and `to` sections anymore.

Reply:
yep, extra output topics are hidden if empty now and cleaners shouldn't have `from` and `to` section